drivers/shared/executor/resource_container_linux.go

```diff
@@ -23,3 +25,46 @@ func (rc *resourceContainerContext) executorCleanup() error {
 	}
 	return nil
 }
+
+func (rc *resourceContainerContext) isEmpty() bool {
+	return rc.groups == nil
+}
+
+func (e *UniversalExecutor) getAllPids() (map[int]*nomadPid, error) {
```

Comment:
I would move method to `executor_universal_linux.go` file as it's a UniversalExecutor linux method.